[2024-12-17T10:09:32.111+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T10:09:32.263+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag_v2.second_task scheduled__2024-06-10T05:00:00+00:00 [queued]>
[2024-12-17T10:09:32.338+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag_v2.second_task scheduled__2024-06-10T05:00:00+00:00 [queued]>
[2024-12-17T10:09:32.342+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-17T10:09:32.410+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): second_task> on 2024-06-10 05:00:00+00:00
[2024-12-17T10:09:32.431+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1114) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-17T10:09:32.436+0000] {standard_task_runner.py:72} INFO - Started process 1149 to run task
[2024-12-17T10:09:32.438+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag_v2', 'second_task', 'scheduled__2024-06-10T05:00:00+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmplqjo7tdc']
[2024-12-17T10:09:32.473+0000] {standard_task_runner.py:105} INFO - Job 26: Subtask second_task
[2024-12-17T10:09:32.698+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag_v2.second_task scheduled__2024-06-10T05:00:00+00:00 [running]> on host 8348eb091483
[2024-12-17T10:09:33.132+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='srush' AIRFLOW_CTX_DAG_ID='my_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-10T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-10T05:00:00+00:00'
[2024-12-17T10:09:33.142+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T10:09:33.215+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-17T10:09:33.229+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo task2 runs after task1']
[2024-12-17T10:09:33.292+0000] {subprocess.py:99} INFO - Output:
[2024-12-17T10:09:33.297+0000] {subprocess.py:106} INFO - task2 runs after task1
[2024-12-17T10:09:33.299+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-17T10:09:33.484+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T10:09:33.488+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_first_dag_v2, task_id=second_task, run_id=scheduled__2024-06-10T05:00:00+00:00, execution_date=20240610T050000, start_date=20241217T100932, end_date=20241217T100933
[2024-12-17T10:09:33.672+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T10:09:33.714+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
